text/0000-filepath-constructor.md

```diff
@@ -0,0 +1,123 @@
+- Feature Name: Split `FilePath` constructor to guarantee constructor with `AmbientAuth`
+- Start Date: 2021-07-02
+- RFC PR:
+- Pony Issue:
+
+# Summary
+
+It should be without error to constructor a `FilePath` when `AmbientAuth` is available. The current constructor uses a union of `(FilePath | AmbientAuth)` to limit capabilities and access for the newly created `FilePath`. Insufficient capabilities or access allows the first half of this union, `FilePath`, to fail, however the latter half `AmbientAuth` can never fail.
```

Comment:
> It should be without error to constructor a `FilePath` when `AmbientAuth` is available.

Definitely a typo in there somewhere.

Reply:
I would say so. Not sure how I did that.